content/blog/2023/08/camel4-whatsnew/index.md

```diff
@@ -0,0 +1,218 @@
+a dependency was downloaded or resolved from a local maven repository. The `camel run` command also has a `--verbose` flag to output
+more details in dependency resolution that can help during troubleshooting.
+
+The `camel-main` runtime now supports exporting with Kubernetes manifest and build support, to make it easier to build
+container images that are ready to run on Kubernetes.
+
+We made many improvements to Camel JBang, making it a great way to experiment with Camel and a useful companion tool for traditional 
+Camel development.
+
+### 8) XML DSL with beans
+
+We have been working on to unify the YAML, XML and Java DSL, so that they have feature parity related to configuring beans. For example,
+in XML DSL (`camel-xml-io`) you can now declare beans and Camel routes in the same XML file with `<camel>` as the root tag:
+
+```xml
+<camel>
+
+    <bean name="greeter" type="com.foo.Greeter">
+        <properties>
+            <property key="message" value="Hello World" />
+        </properties>
+    </bean>
+
+    <route id="my-route">
+        <from uri="direct:start"/>
+        <bean ref="greeter"/>
+        <to uri="mock:finish"/>
+    </route>
+
+</camel>
+```
+
+Then Camel handles the dependency injection, among the `<beans>`. 
+
+You can also use Spring dependency injection (which is more advanced), by inlining Spring `<beans>` tag with the spring namespace.
+See more in the documentation for `camel-xml-io-dsl` module.
+
+There is more work to be done, and we are planning to see if we can also use this to simplify migrating from legacy OSGi Blueprint 
+(and Spring XML files, eg `<beans>`) to the modern Camel DSL.
+
+### 9) New components
+
+- `camel-aws2-step-functions`: Manage and invoke AWS Step functions
+- `camel-azure-files`: Send and receive files on Azure File Storage
+- `camel-dhis2`: Integrate with DHIS2 (health-level)
+- `camel-observation`: Observability using Micrometer Observation
+- `camel-opensearch`: Send requests to OpenSearch
+- `camel-parquet-avro`: Parquet Avro serialization and de-serialization
+- `camel-platform-http-main`: Platform HTTP for Camel Main runtime
+- `camel-yaml-io`: YAML DSL route dumper
+- `camel-zeebe`: Integration with Camunda Zeebe
+
+### 10) Miscellaneous improvements
+
+The Camel maven plugins has been made compatible with Apache Maven 4.
+
+Camel 4 now requires JUnit 5 for unit tests, with the test components that have -junit5 as suffix.
+
+If you work with XML or JSON payloads, then you can log the body in pretty format with:
+
+    .log("${prettyBody}")
+
+And in XML:
+
+    <log message="${prettyBody}"/>
+
+And in YAML:
+
+    - log: "${prettyBody}"
+
+We have added more _dev_console_s that provide insights into your running Camel application, which
+can be used together with Camel JBang and also visible in the developer web console.
+
+### Migrating to Camel 4
+
+Support for OSGi (via Apache Karaf) and `camel-cdi` has been removed. 
```

Comment:
Is it worth mentioning how these technologies will be supported going forward (e.g., that Camel Karaf will continue to evolve as a separate sub-project)?